environment/src/main/java/com/sequenceiq/environment/environment/v1/EnvironmentController.java

```diff
@@ -268,6 +274,15 @@ public void postStopByCrn(@ResourceCrn @TenantAwareParam String crn) {
         environmentStopService.stopByCrn(crn);
     }
 
+    @Override
+    @CheckPermissionByResourceCrn(action = AuthorizationResourceAction.DESCRIBE_ENVIRONMENT)
+    public CredentialResponse verifyCredentialByEnvCrn(@ResourceCrn String crn) {
```

Comment:
do we need this for the /name endpoint ? most of the time we have both /crn and /name

Reply:
We don't need. UI will use the version which works with crn. I can add the /name version for consistency and usability reason if I have to.